Add Perses dashboards and related configurations

ibakshay opened the pull request:
Signed-off-by: Akshay Iyyadurai Balasundaram <akshay.iyyadurai.balasundaram@sap.com>

## Description
<!--
Please do not leave this blank 
This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 
-->

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

<!-- 
Please use this format link issue numbers: Fixes #123
https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
-->
- Related Issue # (issue)
- Closes # (issue)
- Fixes # (issue)

> Remove if not applicable

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] Separate ticket for tests # (issue/pr)

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Added to documentation?

- [ ] 📜 README.md
- [ ] 🤝 Documentation pages updated
- [ ] 🙅 no documentation needed
- [ ] (if applicable) generated OpenAPI docs for CRD changes

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

richardtief commented on charts/kubernetes-operations/perses-dashboards/prometheus-details.json:
```diff
@@ -0,0 +1,1125 @@
+{
+  "kind": "Dashboard",
+  "metadata": {
+    "name": "prometheus-details",
+    "project": "default"
+  },
+  "spec": {
+    "display": {
+      "name": "Prometheus Details"
```
Because I just saw it. The Prometheus Plutono dashboard is integrated into the kube-monitoring chart over there.

https://github.com/cloudoperators/greenhouse-extensions/tree/main/kube-monitoring/charts/dashboards

richardtief commented on charts/kubernetes-operations/perses-dashboards/perses-overview.json:
```diff
@@ -0,0 +1,755 @@
+{
+  "kind": "Dashboard",
+  "metadata": {
+    "name": "perses-overview",
+    "project": "default"
+  },
+  "spec": {
+    "display": {
+      "name": "Perses / Overview"
+    },
+    "panels": {
+      "0_0": {
+        "kind": "Panel",
+        "spec": {
+          "display": {
+            "name": "Perses Stats"
```
This one you can keep within the Perses Plugin.

ibakshay commented:
@richardtief Thank you very much for the suggestion. I have removed both the Perses and Prometheus self-monitoring dashboards from this repository and added directly to the greenhouse-extensions repository in this PR - https://github.com/cloudoperators/greenhouse-extensions/pull/858
